### ISSUE TITLE
ImportError

### ISSUE THREAD

**Issue by Sibiryakanton** [anibali/pywebp#38]
When I try to use the code with a package in the pipeline, I get an error:

```
project_name/modules/image_handler.py:6: in <module>
    import webp
/venv/project-env-9TtSrW0h-py3.10/lib/python3.10/site-packages/webp/__init__.py:6: in <module>
    from _webp import ffi, lib
E   ModuleNotFoundError: No module named '_webp'

```

How can i fix it?

**Comment by anibali** [comment]
How did you install the webp package, what version did you install, and what platform are you on (Windows/Linux)?

**Comment by x2a-io** [comment]
Running into this same issue on MacOS (12.6), Python 3.10.10 - with the package installed via pip: `pip install -U webp`

**Comment by anibali** [comment]
Unfortunately macOS is currently not supported, I've only built wheels for Windows and Linux. In theory it should be able to build from the source distribution, but it looks like that is not working based on your results.

Since I don't have access to Apple hardware it is difficult for me to provide support, but I can try to see if I can build a wheel for macOS and hopefully it just works. I'll give that a go and let you know.

**Comment by x2a-io** [comment]
Ah, I see thanks! I could swear it was working in the past... just not sure how many versions ago though. I'll try and build on my end - but also happy to test whatever ones you build.

**Comment by x2a-io** [comment]
Everything works fine with webp==0.1.4. It seems something changed with 0.1.5

**Comment by Sibiryakanton** [comment]
1. poetry add webp
2. webp = "^0.1.5"
3. Ubuntu 20.04.5 LTS

**Comment by anibali** [comment]
The issue is that installing from the sdist (source tarball) is broken, and this affects all platforms. It seems that the problem started after I moved the project to Poetry. I need to fix the sdist, and then as a bonus I will provide wheels so that it's faster to install on macOS in the future.

**Comment by Sibiryakanton** [comment]
With 0.1.4 i got the other error in lambda:
https://pastebin.com/zkMew4qf

Gone if switched to 0.1.5

**Comment by anibali** [comment]
That was quite a nasty issue to get to the bottom of. It seems like the root cause was a change in Poetry related to build scripts, which broke builds from the sdist tarball: https://github.com/python-poetry/poetry/issues/2740

I've now released a new version, 0.1.6, which should fix the problem. Please give that a go and let me know if it works for you.

**Comment by Sibiryakanton** [comment]
Yep, it works, you are brilliant human being. :)
